Tests/TokamakTests/ViewReactToDataChangesTests.swift

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@testable import TokamakCore
+import XCTest
+import OpenCombineShim
+
+class ViewModifierTests: XCTestCase {
```

Comment:
@j-f1 @carson-katri 
Another thing is, I've added `.onReceive` & `.onChange` modifiers. With some tests here, however, did not have any luck running the tests itself

Reply:
Looks like they perform their setup in `onAppear`, but if the test isn't hooked up to a renderer `onAppear` will never be called. Can you try using the `TestFiberRenderer`?

Reply:
ok